package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@buildium/angular-ui",
-  "version": "1.0.1",
+  "version": "1.0.2-0",
```

Comment:
you probably haven't updated from master @jweisbeck the prerelease should be on version 1.1.0